### ISSUE TITLE
RG option

### ISSUE THREAD

**Issue by jjfarrell** [citiususc/BigBWA#4]
Is there a way to specify the ReadGroup(RG)?  It is needed in the sam/bam file for the GATK pipeline and other genotype calling programs.

From the bwa documentation.....

-R STR  Complete read group header line. ’\t’ can be used in STR and will be converted to a TAB in the output SAM. The read group ID will be attached to every read in the output. An example is ’@RG\tID:foo\tSM:bar’.

**Comment by jmabuin** [comment]
There is not a way for now. But we will try to add this option as soon as possible.

**Comment by jmabuin** [comment]
The option -rg to add a group header line has been added in commit  [9dd7185](https://github.com/citiususc/BigBWA/commit/9dd7185eac45d650ba99bd8714c950e2519bd447) but we could not test it as our cluster is being used right now by other partners.

Please can you check it and provide us some feedback?

**Comment by jjfarrell** [comment]
Sorry I missed  your April 8th message. Has this feature been tested and merged yet?

**Comment by jmabuin** [comment]
I just tested it and solved a small bug in an output file in 79b6152

It works fine, but you need to scape the "\t", for example

```
hadoop jar BigBWA.jar -archives bwa.zip -D mapreduce.input.fileinputformat.split.minsize=15740181 -D mapreduce.input.fileinputformat.split.maxsize=15740181 -algorithm mem -reads paired -rg @RG\\tID:foo\\tSM:bar -r -index /opt/Genomica/BaseHumano/hg38 Genomica/Test.FQBDP ExitTest
```
